interface/themes/style_olive.css

```diff
@@ -1,7 +1,7 @@
 /**
  * Stylesheet for flat colors
```

Comment:
brought this script changes into codebase
(note I removed the end of line whitespaced)